app/routers/guest/meta.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any
+
+from fastapi import APIRouter, HTTPException, status
+
+from app import models
+from app.i18n import _
+
+router = APIRouter()
+
+
+@router.get(
+    "/meta/{domain}",
```

Comment:
Just curious - does the frontend need to do these with individual requests? One request is much faster.

Reply:
Well, I'm thinking of using this to replace the current "options" values in each select where is needed, so I guess yes, we would need individual requests (unless you are thinking of a better approach where we can call this just once and then store the result in the frontend and call that result from everywhere is needed instead)

Reply:
Yes, something like that would be better. I don't write React, but I'm sure it's doable.